test/js/computed-style-property-map.js

```diff
@@ -8,16 +8,9 @@ suite('ComputedStylePropertyMap', function() {
     document.documentElement.removeChild(this.element);
   });
 
-  test('ComputedStylePropertyMap is a ComputedStylePropertyMap and a ' +
-      'StylePropertyMap', function() {
-    var computedStyleMap = new ComputedStylePropertyMap(this.element);
-    assert.instanceOf(computedStyleMap, ComputedStylePropertyMap);
-    assert.instanceOf(computedStyleMap, typedOMTesting.StylePropertyMap);
-  });
-
-  test('ComputedStylePropertyMap.get works for properties that can only be ' +
+  test('Computed StylePropertyMap.get works for properties that can only be ' +
```

Comment:
Wording here is a little odd. Maybe
"Computed StylePropertyMapReadOnly.get works for properties that can only be NumberValues"
or
"Getting opacity from the computed style map works as expected"